code/modules/power/supermatter/supermatter.dm

```diff
@@ -324,6 +324,9 @@
 
 
 /obj/machinery/power/supermatter/Bumped(atom/AM as mob|obj)
+	if(istype(AM,/obj/mecha) || istype(AM,/obj/structure/closet))
+		qdel(AM)
```

Comment:
What does qdelling a closest do to the contents? I see in https://github.com/HippieStationCode/HippieStation13/blob/5462010e5bcc8a61ad5618ffd0f04f07eeb570b0/code/game/objects/structures/crates_lockers/closets.dm#L191 it first empties the contents and then qdels it. Should we do the same here?